packages/components/src/internal/components/labels/PrintLabelsModal.tsx

```diff
@@ -235,19 +238,28 @@ export class PrintLabelsModalImpl extends PureComponent<PrintModalProps & Inject
                     )}
                     <div className="top-spacing">
                         <b>
-                            Label template
+                            Label templates
                             <LabelHelpTip title="BarTender Label Template">
                                 <p>
                                     Provide the label template to use with BarTender. The path should be relative to the
                                     default folder configured in the BarTender web service.
```

Comment:
seems like this tooltip text might be more helpful now on the admin Manage Label Templates form instead of here, since the path can't be changed at this point

Reply:
moved